storm-server/src/main/java/org/apache/storm/daemon/nimbus/Nimbus.java

```diff
@@ -3356,6 +3356,12 @@ public void rebalance(String topoName, RebalanceOptions options)
             if (subject != null) {
                 options.set_principal(subject.getPrincipals().iterator().next().getName());
             }
+            String topoId = toTopoId(topoName);
```

Comment:
I think the change here should not be part of this PR

Reply:
Rebalance will not work without the change below. Should the change to Nimus.java be part of a different Jira then?

Reply:
Reverted Nimbus.java rebalance() changes.

Reply:
Thanks